widgetry_demo/src/lib.rs

```diff
@@ -504,6 +504,11 @@ fn make_controls(ctx: &mut EventCtx) -> Panel {
                         .map(|_| widgetry::Spinner::new(ctx, (0, 11), 1))
                         .collect::<Vec<_>>(),
                 ),
+                Widget::col(
```

Comment:
I'm not sure if you've noticed this part of the demo, but I'm working to keep most controls exactly the same height. 

This has shown to make it easier to get decent layouts by defalut when using a combination of controls.

Reply:
I saw it from one of the last PRs; eventually we'll want to hide the row of controls behind some other tab (perhaps when we have proper support for those :D). In the meantime, same height is definitely a desirable property! I've considered making `Widget::row` automatically put `centered_vert()` on all its members, but that would kind of be the hacky workaround and definitely cause problems in many cases.